### DISCUSSION TITLE
How do you hightlight a column?

### DISCUSSION THREAD

**Discussion by dj-foxxy** [microsoft/monaco-editor#3730] [Q&A] [answered]
I want to highlight column 80 (e.g., make the background colour slightly different) to indicate the line is getting long. How do I do this?

Thanks.

**Comment by tamayika** [comment, marked as answer]
You can do with ruler and custom js.
Please see below example.
https://microsoft.github.io/monaco-editor/playground.html?source=v0.37.0-dev.20230307#XQAAAAJwAwAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscw4QHD83g6N3srnAR_eyMsWabmN_MF6LcQLpPrPvV6g97RHPYml3-V1eDdGqTsIpJefMK6N6wtTVfGUvp-jA_ZTPUz_mvYqHmLED1_KszxuJBLoG154Xb-LH1ziTBJbCIFqshPFZBHkV63zCaZ2BE0hCH-RkerPYXqok1Q7edqkMUChiJyNY9QW8X9oM72LB4rnTaA6VNx57DVxQl6N6UmtXsbTdDM2YnOPW0w0hkPk5ofLltmyvj1eaBvQo1Kf1FJ5dKYkIVEDy4sX6Sdp0xNvPDrIaBVjJivCzbbcAvXit9VH5yAeEaE0LjAoh6ItR-ZjaQL9pQAwUCF1rYJrPLEqoU2YBBlVvEt2d3gfh5VaqcKXy-dJslSqbCed-8pshGkUQ1OoYIxouYxM0ruwvISJZM2mrvYPIRbeRN--KBE3gdhhh0bFQ0xuKhHYyCBpujiRoI_ni17XC8npJUxYLhM9Rg-RLLto99TLxEEGAI04nC0GKTPUF_flrsZu_qVD6KgVuLYlDztdSlzZOfC0jXRjKQGY9wyZSoPF7IfyarSGAtujuxlHARs6rkB-Apk19revLy5gnyGcLWheugqdpc8wxPk6taOQz5w9eV_yf24CP0

**Reply by dj-foxxy** [reply]
Thanks! I didn't know about rulers.
